client/src/utils/context/AvatarContext.js

```diff
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+import PlaceholderAvatar from '../../assets/profile/avatarPlaceholder.svg';
+
+function noop() {}
+const AvatarContext = createContext({
+  image: { PlaceholderAvatar },
```

Comment:
I would not create an additional object here

```suggestion
  image: PlaceholderAvatar,
```